### ISSUE TITLE
include() via browserify-rails 

### ISSUE THREAD

**Issue by alfredoreduarte** [glittershark/reactable#236]
This might come as a noob question.

Using [browserify-rails](https://github.com/browserify-rails/browserify-rails) in my Rails (duh) app. Given that "Reactable also exposes a set of CommonJS modules for piece-by-piece use with Node, Webpack, Browserify, etc. These modules are located in the lib folder at the root of this repositiory.", I've been trying to require Reactable in these ways (not at the same time obviously): 

```
var Reactable = require('./lib/Reactable')
var Reactable = require('/lib/Reactable')
var Reactable = require('lib/Reactable')
var Reactable = require('Reactable')
```

None of those attempts worked.

Browserify-rails saves everything inside node_modules, and I'm already using some other modules, so I don't know what exactly I'm doing wrong here. Any help? Thanks!

**Comment by franleplant** [comment]
Hi @alfredoreduarte 
 tldr

 `var Reactable = require('reactable')` will do the job, assuming that you installed the lib with npm. i.e. `npm install reactable`

In any case, this is not a `Reactable` bug and thus this issue should be closed.

Long answer:

browserify-rails uses browserify, and browserify in turn uses Node's module system. 
For future reference, when trying to require modules installed via `npm`, use the npm identifier.
The npm identifier is the one that you used to install the package i.e. `npm install package` package is the identifier. The identifier it's also on the lib's package.json.


NOTE: if you installed the lib via downloading the files manually or via bower then you are going to require the Lib's file relative to the file you are requiring it from. For further understanding of the way Node's handles its module system see https://nodejs.org/api/modules.html

Thanks for stopping by!
